doc/misc/system-state.7.rst

```diff
@@ -0,0 +1,40 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _systemstate_misc_ref-label:
+
+##############
+ System state
+##############
+
+Description
+===========
+
+The DNF5 system state consists of several TOML files located in the /usr/lib/sysimage/libdnf5/ directory. DNF5 uses the system state to:
```

Comment:
Last time when we spoke about these files with a user, we articulated the files as an implementation details the users should not rely on.

If we want to keep that impression, we should mention that in the top of this page. Something like:

> The way of storing the DNF5 system state is an internal implementation detail that can change any time. To modify the state always use DNF5 command-line interface or DNF5 API.